src/cli.py

```diff
@@ -5,13 +5,14 @@
 
 from src.commands import remove
 from src.commands.install import install
+from src.commands.test.test import test
```

Comment:
Nit: You can reexport `test` in `src/commands/test/__init__.py` to avoid `...test.test`.